clustering/clustering.py

```diff
@@ -0,0 +1,446 @@
+import os, sys
```

Comment:
formatting of imports - see https://www.python.org/dev/peps/pep-0008/#imports about grouping